diff -aur ttf2eot-0.0.2-2/OpenTypeUtilities.h ttf2eot-new/OpenTypeUtilities.h
--- ttf2eot-0.0.2-2/OpenTypeUtilities.h	2009-04-30 11:18:46.000000000 +0200
+++ ttf2eot-new/OpenTypeUtilities.h	2011-10-14 17:15:09.982065512 +0200
@@ -28,6 +28,8 @@
 #ifndef OpenTypeUtilities_h
 #define OpenTypeUtilities_h
 
+#include <cstddef>
+
 using std::vector;
 
 bool getEOTHeader(unsigned char * fontData, size_t fontSize, vector<uint8_t>& eotHeader, size_t& overlayDst, size_t& overlaySrc, size_t& overlayLength);
